src/main/res/values/strings.xml

```diff
@@ -181,5 +181,18 @@
     <string name="pause_the_audio" translatable="false">"Pause the audio"</string>
     <string name="upload" translatable="false">"Upload"</string>
     <string name="sign_in" translatable="false">"Sign in"</string>
+    <string name="upload_channel" translatable="false">File Upload</string>
```

Comment:
There seems to be an error in the upload_channel string value according to the travis build. Though, I don't see any error here in the file.

Reply:
Yeah, There is no error. @csoni111 can you help me?

Reply:
@shivammaindola The string `upload_channel` is already present in the current beta branch https://github.com/amahi/android/blob/beta/src/main/res/values/strings.xml#L185 , hence it getting duplicated here.
Merge latest beta branch in your local branch and you will get the error.
Also why have you added these strings here? It is not related to the functionality you are implementing.

Reply:
Yes, I forgot to rebase the beta branch. Sorry, I'll fix it.